src/flavors/eu/components/ecl-footers-eu/README.md

```diff
@@ -1,4 +1,4 @@
-# Footers
+# EU Footers
```

Comment:
When sharing preview with design team, a request for updated specification would possibly make the documentation actually useful.